das-cli/src/common/systemutils/sys_info.py

```
@@ -21,7 +21,7 @@ def __init__(self):
     )
 
     def get_cpu_info(self):
-        cpuUsagePercent = psutil.cpu_percent()
+        cpuUsagePercent = psutil.cpu_percent(interval=2)
```

Pylint (4.0.6)

[convention] Variable name "cpuUsagePercent" doesn't conform to '(([a-z][a-z0-9_]{0,})|(_[a-z0-9_]*))$' pattern

(C0103)